### PR TITLE
eliminated redundant calls

### PR DESCRIPTION
**Pull request by pinkysnowman**
eliminated redundant calls for the modpath :3

### CODE REVIEW

**Comment by paramat** [comment]
:+1:

File: mods/default/init.lua [hunk 1]
```diff
@@ -6,6 +6,7 @@
 -- Definitions made by this mod that other mods can use too
 default = {}
 
+default.path = minetest.get_modpath("default")
```
Comment (t4im) [inline mods/default/init.lua]: You could make the path local. Other mods can still use `minetest.get_modpath("default")` if they need it, so there is no reason to make it globally available. :)

**Comment by sfan5** [comment]
:+1: though the suggestion by @t4im is a good idea

**Comment by pinkysnowman** [comment]
@t4im done :3

**Comment by rubenwardy** [comment]
:+1: changes are good

**Comment by paramat** [comment]
5e9e3f7e84fafd6ece535b1c7aff3b9ea4e6405b

**Pull request closed without merging**